.gitignore

```diff
@@ -1,2 +1,3 @@
 # Main project configuration file.
 /config
+/api-key
```

Comment:
We should be safe to just remove this now, since neither of us have this file in our local repositories.